### PR TITLE
Update alternative spec to be considered working draft

### PR DESCRIPTION
**Pull request by RupertBenWiser**
Making it clear from the top level README.md that this represents the current working prototype for Chromium in particular.

The goal is to help make it clear that we've thought about the details of how this could work to any community group that we present this to for adoption.

### CODE REVIEW

File: docs/spec/index.bs [hunk 1]
```diff
@@ -45,12 +45,21 @@ Markup Shorthands: markdown yes, css no
...
-# Web Environment Integrity API # {#api}
+# Key terms # {#key-terms}
 
-<i>Todo</i>
+The <dfn for="web environment">web environment</dfn> is defined as <i>TODO</i>
 
 # Attesters # {#attester}
 
+The term <dfn for="attester">attester</dfn> refers to a third party capable of returning an [=Integrity verdict=]. A <dfn for="Integrity verdict">Integrity verdict</dfn> refers
+to an response that confirms if the [=attester=] trusts the [=web environment=] the [=user agent=] is
```
Comment (rayankans) [inline docs/spec/index.bs:55]: nit: a response

**Comment by RupertBenWiser** [comment]
I should have done a `git mv`. Sorry about that! To save time, I just copied the alternatives to the index directories. Those are unchanged.

File: docs/spec/index.bs [hunk 2]
```diff
@@ -59,6 +68,63 @@ Markup Shorthands: markdown yes, css no
...
+    1. Return |promise|
+</div>
+
+## {{EnvironmentIntegrity}} ## {#environment-integrity}
+
+<xmp class="idl">
+  interface EnvironmentIntegrity {
+    readonly attribute ArrayBuffer attestationToken;
+    DOMString encode();
+    [Default] object toJSON();
```
Comment (rayankans) [inline docs/spec/index.bs:117]: drop the default

File: docs/spec/index.bs [hunk 3]
```diff
@@ -59,6 +68,63 @@ Markup Shorthands: markdown yes, css no
...
+  };
+</xmp>
+
+: attestationToken
+:: The attestation token is a COSE signed CBOR object as an ArrayBuffer from the attester.
+: encode()
+:: The encode method will return a Base64 string representation of the attestation token.
+: toJSON()
+:: The toJSON method returns a JSON representation of the attestation token. Useful for local debugging.
```
Comment (rayankans) [inline docs/spec/index.bs:126]: i'd mention that this is human readable after the CBOR payload has been decoded

**Review by rayankans — APPROVED** [review]

File: docs/spec/index.bs (hunk 1 above)
Comment (RupertBenWiser) [reply docs/spec/index.bs]: Updated.

File: docs/spec/index.bs (hunk 3 above)
Comment (RupertBenWiser) [reply docs/spec/index.bs]: Makes sense!

File: docs/spec/index.bs (hunk 2 above)
Comment (RupertBenWiser) [reply docs/spec/index.bs]: Removed. Don't know how this stuck around.

**Pull request merged**